static/js/tippyjs.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+import tippy from 'tippy.js';
+import {delegate, hideAll} from 'tippy.js';
+window.tippy = tippy;
```

Comment:
I know this is still a draft, but we shouldn’t be adding any further detritus to `window`—just import it everywhere it’s used.

Reply:
Okay, thanks!